### ISSUE TITLE
OrderBy on RelatableQuery Bug

### ISSUE THREAD

**Issue by latwelve** [laravel/nova-issues#3408]
Latest versions of Nova and PHP8, however this has been around for a while:
https://github.com/laravel/nova-issues/issues/1484

The issue:
Its not possible to modify a relatableQuery with orderBy() as it gets overwritten by nova core code.

I'd like the following code to respect orderBy so that we can have more flexibility on how we display a BelongsTo select on a resource.

```
public static function relatableQuery(NovaRequest $request, $query) { 
    return $query->orderBy('display_order','asc');
}
```

**Comment by davidhemphill** [comment]
👋 Hi there! In order to keep this repository focused on bug reports, we auto-close feature requests and requests for help. Feel free to post your feature requests so others can discuss and add reactions. We'll keep an eye on them for later planning.

**Comment by crynobone** [comment]
Duplicate of #3272 and a PR has been pushed pending review.

**Comment by crynobone** [comment]
PR merged, you should be able to do the following after next patch release:

```php
BelongsTo::make('User')->dontReorderAssociatables(),
```

**Comment by crynobone** [comment]
This has been fixed in [v3.26.0](https://nova.laravel.com/releases/3.26.0)
